smithy.yaml

```diff
@@ -1,8 +1,7 @@
 project: dart
 language: dart
 
-# dart 1.24.2
-runner_image: drydock-prod.workiva.net/workiva/smithy-runner-generator:203768
+runner_image: google/dart:1.24.3
```

Comment:
may as well delete this file.. everything has been building on Workiva Build via the Dockerfile for quite some time
(I suspect Rosie is failing her dependency scan because of this not being an *allowed* Smithy runner image)